config/samples/go_example_loadtest.yaml

```diff
@@ -27,9 +27,14 @@ spec:
     language: go
     name: '0'
     run:
+      args:
+      - -c
+      - |
+        timeout --kill-after="$${KILL_AFTER}" "$${POD_TIMEOUT}" \
```

Comment:
in this file, I can still see `$$` (they're gone in the other files).

Reply:
I was experimenting with go, fix it, thank you for pointing it out!